src/directives/mwlCalendarDay.js

```diff
@@ -105,7 +109,8 @@ angular
         dayViewEventWidth: '=',
         customTemplateUrls: '=?',
         cellModifier: '=',
-        templateScope: '='
+        templateScope: '=',
+        timePosition: '=' 
```

Comment:
Travis is failing because of a trailing space at the end of this line, can you fix it please?